src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -53,4 +53,24 @@
         public static final String CREDIT = "CR";
         public static final String DEBIT = "DR";
```

Comment:
I'm also going to need this debit/credit codes for the Collector processing. It might make sense to move them into the top-level class or to at least rename the nested class.